packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -20,16 +20,21 @@ async function promptForConfig() {
     {
       type: 'checkbox',
       name: 'additionalFeatures',
-      message: 'What additional features does your app require',
+      message: 'What additional features does your app require (these are checkboxes)',
+      default: ['electric-flow', 'header-footer'],
```

Comment:
I'm not sure we should default electric-flow to on

Reply:
I do not know if this is true, but I was under the assumption that what Tyler did with the HF templating expects EF to be true/on as well.

Tyler would have to chime in further though.